hazelcast/src/test/java/com/hazelcast/map/impl/mapstore/MapStoreTest.java

```diff
@@ -137,22 +137,20 @@ public String load(String key) {
         assertFalse(loadCalled.get());
     }
 
-    @Test(timeout = 120000)
+    @Test(expected = NullPointerException.class)
```

Comment:
Same comment as above with the `ExpectedException` rule.

Reply:
addressed via ExpectedException rule.